### DISCUSSION TITLE
Doğum tarihi nedir?

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#247] [Zeka Sorusu]
Ahmet ve Bilal, henüz yeni arkadaş oldukları Cansunun doğum gününün ne zaman olduğunu bilmek istiyorlar.
Cansu onlara 10 muhtemel tarihin olduğu bir liste veriyor.
15 Mayıs - 16 Mayıs - 19 Mayıs
17 Haziran - 18 Haziran
14 Temmuz - 16 Temmuz
14 Ağustos - 15 Ağustos - 17 Ağustos
Bundan sonra Cansu, Ahmet’e sadece doğduğu ayı, Bilal’e ise sadece doğduğu günü söylüyor. Sonrasında şu diyalog yaşanıyor:
• Ahmet: Cansunun doğum tarihini bilmiyorum; fakat Bilal’in da bilmediğini biliyorum.
• Bilal: Başta Cansunun doğum tarihini bilmiyordum; fakat şimdi artık biliyorum.
• Ahmet: O zaman ben de Cansunun doğum tarihini biliyorum.
Bu durumda Cansunun doğum tarihi nedir?

**Comment by HabilCcabbar** [comment]
Ahmet'in söylediği ifadeden Cansu'nun doğum ayının tekrar eden bir ay olduğunu anlıyoruz. ("Cansu'nun doğum tarihini **bilmiyorum**.")
 Bilal'in söylediği ifadeden de Cansu'nun doğum gününün tekrar ettiğini anlıyoruz. ("Başta Cansu'nun doğum tarihini **bilmiyordum**.") 
 Yani Cansu'nun doğum tarihinin hem ayı hem de günü tekrar etmeli. **19 Mayıs ve 18 Temmuz elenir**.
 Ahmet ayrıca Bilal'in bilmediğinden emin en başta. Yani doğum ayı Mayıs veya Haziran olamaz. Çünkü o ayda tekrar etmeyen günler var ve bu Bilal'in kesin bildiği ihtimalini doğuruyor. Yani **15 Mayıs, 16 Mayıs ve 17 Haziran elenir**.
 Bilal, Ahmet'in ifadesinden sonra tarihi çözüyor yani kalan 5 olasılıktan tekrar edenler olamaz. **14 Temmuz ve 14 Ağustos elenir**.
 Bundan sonra Ahmet de çözüyor yani kalan üç ihtimalden **15 Ağustos ve 17 Ağustos elenir** çünkü aynı aylar. Yani cevap: **16 TEMMUZ**
